rules/coding-style/src/Rector/Throw_/AnnotateThrowablesRector.php

```diff
@@ -125,14 +125,32 @@ private function identifyThrownThrowables(Throw_ $throw): array
         }
 
         if ($throw->expr instanceof StaticCall) {
-            return $this->identifyThrownThrowablesInStaticCall();
+            return $this->identifyThrownThrowablesInStaticCall($throw);
         }
 
         return [];
     }
 
-    private function identifyThrownThrowablesInStaticCall(): array
+    private function identifyThrownThrowablesInStaticCall(Throw_ $throw): array
     {
+        dumpf(null, $this, $throw);
+        // 1. Getting the DocBlock of the method using reflection
+        $thrownClass = $throw->expr->class;
+        $classFqn = implode('\\', $thrownClass->parts);
+        $methodNode = $thrownClass->getAttribute('nextNode');
+        $methodName = $methodNode->name;
+        $reflectedMethod = new \ReflectionMethod($classFqn, $methodName);
+        $methodDocblock = $reflectedMethod->getDocComment();
```

Comment:
You can do something like this:

https://github.com/symplify/symplify/blob/eb4cfd2e17b564acf8ab3840c9207f470511c1ca/packages/autowire-array-parameter/src/DependencyInjection/CompilerPass/AutowireArrayParameterCompilerPass.php#L170-L186

Reply:
I think this will work... I'll try...

Reply:
🎉It works like a charm! 🎉

I'm locally running the checks and then I'll update the PR.